docs/testing.md

```diff
@@ -0,0 +1,68 @@
+# Testing
+
+All testing methods described here work in both the root directory and individual package directories.
+
+## Test all the things
+
+`> npm run test`
```

Comment:
I dislike starting with the “>“, as that makes it more difficult to copy/paste. I think the `npm` is a strong enough signal that these are shell commands.